packages/lab/src/dialog/DialogContent.css

```
@@ -1,13 +1,14 @@
 .saltDialogContent {
   color: var(--salt-content-primary-foreground);
-  padding-left: var(--salt-spacing-300);
-  padding-right: var(--salt-spacing-300);
-  flex: 1 1 auto;
+  display: flex;
+  flex-direction: column;
   min-height: var(--salt-text-lineHeight);
-  overflow-y: auto;
+  overflow-y: scroll;
+  background: var(--salt-container-primary-background);
 
-  font-family: var(--salt-text-fontFamily);
-  font-size: var(--salt-text-fontSize);
-  font-weight: var(--salt-text-fontWeight);
-  line-height: var(--salt-text-lineHeight);
+  margin-left: var(--salt-spacing-300);
+  margin-right: var(--salt-spacing-300);
+  padding-right: var(--salt-spacing-300);
+  padding-bottom: 10px;
+  border-bottom: 1px solid var(--salt-separable-tertiary-borderColor);
```

I'm assuming this should use the tokens from the theme?

```suggestion
  border-bottom: var(--salt-size-border) var(--salt-separable-borderStyle) var(--salt-separable-tertiary-borderColor);
```